packages/actor-rdf-parse-html-script/test/ActorRdfParseHtmlScript-test.ts

```diff
@@ -0,0 +1,220 @@
+            quad('http://example.org/a', 'http://example.org/d', 'http://example.org/e'),
+          ]));
+      });
+
+      it('should run with one JSON-LD script and one text/plain script', () => {
+        parseAction.input = stringToStream(
+          `<script type="application/ld+json">
+          [{
+              "@id": "http://example.org/a",
+              "http://example.org/b": "http://example.org/c",
+              "http://example.org/d": "http://example.org/e"
+          }]
+          </script>
+          <script type="text/plain">
+          [{
+              "@id": "http://example.org/a",
+              "http://example.org/b": "http://example.org/c",
+              "http://example.org/d": "http://example.org/e"
+          }]
+          </script>`);
+
+        return actor.run(
+          { context: parseAction.context, handle: { input: parseAction.input }, handleMediaType: 'text/html' })
+          .then(async (output) => expect(await arrayifyStream(output.handle.quads)).toEqualRdfQuadArray([
+            quad('http://example.org/a', 'http://example.org/b', 'http://example.org/c'),
+            quad('http://example.org/a', 'http://example.org/d', 'http://example.org/e'),
+          ]));
+      });
+
+      it('should run with two JSON-LD scripts and one N-Quads script', () => {
+        parseAction.input = stringToStream(
+          `<script type="application/ld+json">
+          [{
+              "@id": "http://example.org/a",
+              "http://example.org/b": "http://example.org/c",
+              "http://example.org/d": "http://example.org/e"
+          }]
+          </script>
+          <script type="application/n-quads">
+            http://example.org/a http://example.org/b http://example.org/c "" .
+            http://example.org/a http://example.org/d http://example.org/e "" .
+          </script>
+          <script type="application/ld+json">
+            [{
+              "@id": "http://example.org/a",
+              "http://example.org/b": "http://example.org/c",
+              "http://example.org/d": "http://example.org/e"
+            }]
+          </script>`);
+
+        return actor.run(
+          { context: parseAction.context, handle: { input: parseAction.input }, handleMediaType: 'text/html' })
+          .then(async (output) => expect(await arrayifyStream(output.handle.quads)).toEqualRdfQuadArray([
+            quad("http://example.org/a", "http://example.org/b", "http://example.org/c", ""),
+            quad("http://example.org/a", "http://example.org/d", "http://example.org/e", ""),
+            quad("http://example.org/a", "http://example.org/b", "http://example.org/c", ""),
+            quad("http://example.org/a", "http://example.org/d", "http://example.org/e", ""),
+            quad("http://example.org/a", "http://example.org/b", "http://example.org/c", ""),
+            quad("http://example.org/a", "http://example.org/d", "http://example.org/e", ""),
+          ]));
+      });
+
```

Comment:
Could you also add a test with a script tag that has _no_ `type` attribute?

Also, a test with a script tag inside a realistic HTML document would be good as well. (with stuff like head, body, h1, ...)